Content.Server/_Goobstation/Devil/Contract/DevilContractSystem.ContractEvents.cs

```diff
@@ -0,0 +1,105 @@
+// SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+// SPDX-FileCopyrightText: 2025 Solstice <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 SolsticeOfTheWinter <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 gluesniffler <159397573+gluesniffler@users.noreply.github.com>
+// SPDX-FileCopyrightText: 2025 gluesniffler <linebarrelerenthusiast@gmail.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using System.Linq;
+using Content.Shared._Goobstation.Devil;
+using Content.Server.Body.Components;
+using Content.Shared.Body.Components;
+using Content.Shared.Body.Part;
+using Robust.Shared.Random;
+
+namespace Content.Server._Goobstation.Devil.Contract;
+
+public sealed partial class DevilContractSystem
+{
+    private void InitializeSpecialActions()
+    {
+        SubscribeLocalEvent<DevilContractSoulOwnershipEvent>(OnSoulOwnership);
+        SubscribeLocalEvent<DevilContractLoseHandEvent>(OnLoseHand);
+        SubscribeLocalEvent<DevilContractLoseLegEvent>(OnLoseLeg);
+        SubscribeLocalEvent<DevilContractLoseOrganEvent>(OnLoseOrgan);
+        SubscribeLocalEvent<DevilContractChanceEvent>(OnChance);
+    }
+    private void OnSoulOwnership(DevilContractSoulOwnershipEvent args)
```

Comment:
Blank line between methods
```suggestion
    }

    private void OnSoulOwnership(DevilContractSoulOwnershipEvent args)
```